src/zenml/integrations/integration.py

```diff
@@ -14,9 +14,11 @@
 from typing import Any, Dict, List, Tuple, Type, cast
 
 import pkg_resources
+import shutil
 
 from zenml.integrations.registry import integration_registry
 from zenml.logger import get_logger
+from zenml.exceptions import DoesNotExistException
```

Comment:
This is the order of imports you would get from running  format.sh that @htahir1 was suggesting. Adding it here to save you some time :)

```suggestion
import shutil
from typing import Any, Dict, List, Tuple, Type, cast

import pkg_resources

from zenml.exceptions import DoesNotExistException
from zenml.integrations.registry import integration_registry
from zenml.logger import get_logger
```